src/components/Dashboard/ThisWeek/NewsletterMenu/MenuItems/ExportPhysical/ExportPhysical.tsx

```diff
@@ -31,8 +40,36 @@ const CloseButton = styled(IconButton)(({ theme }) => ({
   },
 }));
 
-const ExportPhysical = ({ handleClose }: Props): ReactElement<Props> => {
+const LabelButton = styled(Button)(({ theme }) => ({
+  width: '90%',
+  margin: theme.spacing(1, 0),
+}));
+
+const ExportPhysical = ({
+  handleClose,
+  accountListId,
+}: Props): ReactElement<Props> => {
   const { t } = useTranslation();
+  const [session] = useSession();
+
+  const [createExportedContacts] = useCreateExportedContactsMutation({
+    onCompleted: (data) => {
+      window.location.replace(`${data.exportContacts}${session?.user.token}`);
```

Comment:
Would this be better in a new window? Or does it behave fine?

Reply:
Yeah it works fine, visually it doesn't do anything except start the download